lib/util/findImportCSSFromWithStylesImportDeclaration.js

```diff
@@ -0,0 +1,31 @@
+function findImportCSSFromWithStylesImportDeclaration(node) {
+  if (!node.source.value.endsWith('withStyles')) {
+    return null;
+  }
+
+  if (!node.specifiers.length) {
+    // import 'foo';
+    return null;
+  }
+
+  let name = null;
```

Comment:
since this isn't being transpiled, the file needs to be explicitly in strict mode